src/psyclone/tests/domain/lfric/kernel_interface_test.py

```diff
@@ -811,7 +816,8 @@ def test_quad_rule_xyoz():
     assert weightsh_symbol.shape[0] is nqph_symbol
     # weights_v declared and added to argument list
     weightsv_symbol = kernel_interface._symbol_table.lookup("weights_v")
```

Comment:
Should be "weights_z"

Reply:
Yes, already updated due to your previous comment.